src/composeapp.cc

```diff
@@ -36,6 +36,7 @@ bool ComposeApp::up(bool no_start) {
 }
 
 bool ComposeApp::start() { return cmd_streaming(compose_ + "start"); }
+bool ComposeApp::stop() { return cmd_streaming(compose_ + "stop"); }
```

Comment:
this seems to be unused?

Reply:
yes, removed and updated PR via forced push.